repose-aggregator/tests/functional-tests/src/integrationTest/groovy/features/core/configloadingandreloading/StartWithMissingConfigsTest.groovy

```diff
@@ -55,14 +55,14 @@ class StartWithMissingConfigsTest extends ReposeValveTest {
                 waitOnJmxAfterStarting: false)
         repose.waitForNon500FromUrl(reposeEndpoint)
 
-
+//todo: do something with this test?
```

Comment:
Do we want to do something with this test now? A quick refactor would allow us to drop the `@Unroll` and even the`@Category(Slow.class)` if we wanted to. However, this test only tells us the behavior of Repose when missing a metrics config, ignoring all of the other configs we rely on. So that's not terrific. And for that matter, all of these config tests only test a subset of all of our configs (perhaps just because we haven't kept them up-to-date.).

Reply:
I already updated it, i forgot to drop the todo. i swapped it to metrics so that it would continue working.

Reply:
Ah, I see. Fair enough.

The test could be cleaned up a bit, or expanded, but that it outside of the scope of this story.